x-pack/test/functional/apps/ml/anomaly_detection_jobs/job_wizard_validation.ts

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { FtrProviderContext } from '../../../ftr_provider_context';
+import type { FieldStatsType } from '../common/types';
+
+export default function ({ getService }: FtrProviderContext) {
+  const config = getService('config');
+  const esNode = config.get('esTestCluster.ccs')
+    ? getService('remoteEsArchiver' as 'esArchiver')
+    : getService('esArchiver');
+  const ml = getService('ml');
+
+  const jobId = `fq_single_1_${Date.now()}`;
+  const jobGroups = ['automated', 'farequote', 'single-metric'];
+  const aggAndFieldIdentifier = 'Mean(responsetime)';
+  const bucketSpan = '30m';
+
+  const remoteName = 'ftr-remote:';
+  const esIndexPatternName = 'ft_farequote';
+  const esIndexPatternString = config.get('esTestCluster.ccs')
+    ? remoteName + esIndexPatternName
+    : esIndexPatternName;
+
+  const fieldStatsEntries = [
+    {
+      fieldName: '@version.keyword',
+      type: 'keyword' as FieldStatsType,
+      expectedValues: ['1'],
+    },
+  ];
+
+  describe('job wizard validation', function () {
```

Comment:
I reckon you should not create a dedicated test suite for that, and go through all the wizard steps again. 
We already have multiple tests for job wizards. Simply add a test suite to assert job validation. 

For instance in the multi-metric wizards tests: 
```typescript
      await ml.testExecution.logTestStep('job creation displays the validation step');
      await ml.jobWizardCommon.advanceToValidationSection();
```

Add your assertion after these lines

Reply:
@darnautov One of the reasons I created this is not have a nested suite.  
I'll see if I can fit these in elsewhere without a nested suite.